tests/acceptance/features/bootstrap/SpacesContext.php

```diff
@@ -246,7 +246,11 @@ public function getSpaceByName(string $user, string $spaceName): array {
 		if ($spaceName === "Personal") {
 			$spaceName = $this->featureContext->getUserDisplayName($user);
 		}
-		$this->theUserListsAllHisAvailableSpacesUsingTheGraphApi($user);
+		if (strtolower($user) === 'admin') {
```

Comment:
I think I would rather put that switch into an parameter, otherwise we will be very surprised at some point why the admin reacts differently that other users

Reply:
Thanks for suggest, but I think that is correct place where we need check admin user role (maybe in feature change: if user has "Space manager" role). 
admin is not used in tests (he only grants rights). Now I want that admin can get all spaces and delete them.
Web use theUserListsAllHisAvailableSpacesUsingTheGraphApi `graph/v1.0/me/drives` and all tests use this method

if I add parameter, each method(that use `getSpaceByName`) need define who is user (admin or not) 

I may be wrong, but so far I see it this way

Reply:
IMO we could use this as a workaround.